examples/raw_input/RawInput.cpp

```diff
@@ -0,0 +1,82 @@
+////////////////////////////////////////////////////////////
+// Headers
+////////////////////////////////////////////////////////////
+#include <SFML/Graphics.hpp>
+
+#include <vector>
+
+
+namespace
+{
+std::filesystem::path resourcesDir()
+{
+#ifdef SFML_SYSTEM_IOS
+    return "";
+#else
+    return "resources";
+#endif
+}
+} // namespace
+
+
+////////////////////////////////////////////////////////////
+/// Entry point of application
+///
+/// \return Application exit code
+///
+////////////////////////////////////////////////////////////
+int main()
+{
+    // Create the main window
+    sf::RenderWindow window(sf::VideoMode({800u, 600u}), "SFML Raw Mouse Input", sf::Style::Titlebar | sf::Style::Close);
+    window.setVerticalSyncEnabled(true);
+
+    // Load the application font and pass it to the Effect class
+    const auto font = sf::Font::loadFromFile(resourcesDir() / "tuffy.ttf").value();
+
+    auto vectorToString = [](const auto& vector)
+    { return '(' + std::to_string(vector.x) + ", " + std::to_string(vector.y) + ')'; };
+
+    // Create the mouse position text
+    sf::Text mousePosition(font, "Mouse Position: " + vectorToString(sf::Mouse::getPosition()), 20);
+    mousePosition.setPosition({400.f, 300.f});
+    mousePosition.setFillColor(sf::Color::White);
+
+    // Create the mouse raw movement text
+    sf::Text mouseRawMovement(font, "", 20);
+    mouseRawMovement.setFillColor(sf::Color::White);
+
+    std::vector<sf::String> log;
+
+    while (window.isOpen())
+    {
+        while (const auto event = window.pollEvent())
+        {
+            if (event.is<sf::Event::Closed>())
+                window.close();
+
+            if (const auto* const mouseMoved = event.getIf<sf::Event::MouseMoved>(); mouseMoved)
+                mousePosition.setString("Mouse Position: " + vectorToString(mouseMoved->position));
+
+            if (const auto* const mouseMovedRaw = event.getIf<sf::Event::MouseMovedRaw>(); mouseMovedRaw)
+            {
+                log.emplace_back("Mouse Movement: " + vectorToString(mouseMovedRaw->delta));
+
+                if (log.size() > 24u)
+                    log.erase(log.begin());
+            }
+        }
+
+        window.clear();
+        window.draw(mousePosition);
+
+        for (auto i = 0u; i < log.size(); ++i)
```

Comment:
```suggestion
        for (std::size_t i = 0; i < log.size(); ++i)
```

Not a big deal, but `0u` is `unsigned int` which does not match `log.size()`'s type. `std::size_t` does.